example/cnn_text_classification/text_cnn.py

```diff
@@ -238,18 +238,18 @@ def train_without_pretrained_embedding():
     # split train/dev set
     x_train, x_dev = x_shuffled[:-1000], x_shuffled[-1000:]
     y_train, y_dev = y_shuffled[:-1000], y_shuffled[-1000:]
-    print 'Train/Dev split: %d/%d' % (len(y_train), len(y_dev))
-    print 'train shape:', x_train.shape
-    print 'dev shape:', x_dev.shape
-    print 'vocab_size', vocab_size
+    print('Train/Dev split: %d/%d' % (len(y_train), len(y_dev)))
+    print('train shape:', x_train.shape)
+    print('dev shape:', x_dev.shape)
+    print('vocab_size', vocab_size)
    
-    batch_size = 50
+    batch_size = 4
```

Comment:
By setting batch_size = 50, it will cause GPU with less mem(say 1GB) to result in OOM. I have changed it back. No worries.